addon-test-support/@ember/test-helpers/index.ts

```diff
@@ -22,6 +22,7 @@ export { default as settled, isSettled, getSettledState } from './settled';
 export { default as waitUntil } from './wait-until';
 export { default as validateErrorHandler } from './validate-error-handler';
 export { default as setupOnerror, resetOnerror } from './setup-onerror';
+export { default as DebugInfo, getDebugInfo } from './-internal/debug-info';
```

Comment:
Do we need to expose `DebugInfo` here? I would think that constructing the class directly is “private”.

Reply:
Oh you're right. We don't need to instantiate it in `ember-qunit` anymore. I'll remove this.